### ISSUE TITLE
long slow feeds lose axis distance reference

### ISSUE THREAD

**Issue by kwince** [buildbotics/bbctrl-firmware#258]
When feeding at 7IPM at about 16 inches on my 36" X axis suddenly the display will go to negative. 14".

Version 0.4.11
I just boot up, zero the machine
zero it
go to the MDI
execute
```
g0 x0.5 y1.5
g0 z7.25
s17000
f7
g1 y1.25
; i may accientally type in g1 18 here instead of g1 x18, but I do follow it with g1 x18 and it starts to move
g1 x18
;at x=16 the display switches to x=-14 or so
```
this is not a program, so I will try it on the machine with the latest version of the firmware next week.

**Comment by kwince** [comment]
I will break it up and see if it works, probably will. I think this is some sort of overflow in math issue or a timeout issue. I will report about the divided up path and trying it on versions 0.4.12/13 versions. It would be nice if it's a timeout issue if the machine just stopped and the display flashed, or something less harmful than continuing with a negative value.

**Comment by DougCoffland** [comment]
Another customer has experienced this problem. In this case, the user has a very slow feed rate causing a move that takes about 10 minutes. Here are the first few lines of his program:

G20 (inches)

M6 T1 (Change Tool: Diameter: 0.4375 in)

G0Z0.2500

G0X-5.7313Y-0.2188 (origin)

g0z-.63

G1X-5.2402Y-0.2188Z-0.6500F0.1 (origin 1)
G1X-5.7313Y-0.2188f4 (Origin 0)
G0X-5.2402Y-0.2188 (origin 1)

The line that reads G1X-5.2402Y-0.2188Z-0.6500F0.1 (origin 1) causes the move of about .5" on the X axis at a feed rate of .1 inches per minute. About half way through the move, the axis suddenly jumps and loses position. It happens every time.

**Comment by kwince** [comment]
There is apparently a loop or move timeout, I believe.

I can figure it out if I split the move into a multiple moves and see what
happens. Won't be able to do that till next weekend though

**Comment by mmcmilli** [comment]
Copied from my post on the BB Tech Forum:

"I was doing some fly cutting at 2IPM using MDI and noticed some unusual behavior. This behavior was repeatable from any location on the table and when run from a program. I only experimented in x axis, but this behavior may repeat for the others.
My program code was: "G01 G90 X-6.0 F2.0"
After travelling beyond -4.0" the machine would hick-up as if it lost steps and the position readout would flip sign to +4.**" It would continue feeding trying to get to -6" but would repeat the hickup and sign flip when it reached -4.**" again. It would continue this indefinitely. It also occured if I tried to go to +6" in my g-code but with all signs reversed.
After trying several things without affect, I found that if I changed the feed this fault would happen at different locations.
At 1IPM it was ~2"
At 2IPM it was ~4"
At 3IPM it was ~6"
At 4IPM it was ~8"
The pattern is approximately 2xfeed...on my machine.

I was able to work around this by breaking the long travel segment up into smaller chuncks or by using a feed that was higher. I think though, that long linear moves on larger machines could run into this problem at higher feeds based on the pattern here.

Things I tried that did NOT change the behavior.
Changing microstepping.
Changing Acceleration.
Changing Jerk.
Disabling limit switches.
Updating firmware from .4* to most current (New layout and features look awesome btw!)

I also didn't try any mult axis moves or arc moves to see if the problem occurred there. I mention it because others may be having this issue and need a work around and maybe it's something fixable in the code. My lead on the ballscrew would affect the distance travelled so I'll report that too (2.5mm).

Cheers!"

**Comment by DougCoffland** [comment]
I have duplicated this problem. It happens exactly as described.

I simply set my travel per revolution on the X axis to 2.5 mm and ran the exact command that was described.

After it failed, I took looked at the log file and it appears to reverse mid-stream for no apparent reason. See below. Notice that it turns around at -110.952 mm, which is just about 4.37 inches.
**************************
I:Planner:set(#12, line, 1)
I:Planner:Cmd:{"max-vel": 50.8, "max-jerk": 150000000.0, "entry-vel": 0.0, "max-accel": 10000000.0, "type": "line", "first": true, "exit-vel": 0.0, "id": 13, "times": [34.92, 0.0, 34.92, 179930.17, 34.92, 0.0, 34.92], "target": {"x": -152.4}}
I:Comm:< #id=13\nlAAAAAAgJYYSwGA0PTQxZmYYww0Do4YOg2Do4YOg37uw/QA4Do4YOg6Do4YOg
I:Planner:Cmd:{"type": "end", "id": 14}
I:Comm:< #id=14\n
I:Comm:> {"0df":192,"xp":-0.079,"id":13,"v":0.051,"if":0.02,"xx":"RUNNING","xc":17}
I:Comm:> {"xp":-0.096}
I:Comm:> {"xp":-0.099}
I:Comm:> {"xp":-0.319}
I:Comm:> {"xp":-0.539}
I:Comm:> {"xp":-0.76}
I:Comm:> {"xp":-0.98}
I:Comm:> {"xp":-1.2}
I:Comm:> {"xp":-1.42}
I:Comm:> {"xp":-1.447}
I:Comm:> {"xp":-1.454}
I:Comm:> {"xp":-1.671}
I:Comm:> {"xp":-1.891}
I:Comm:> {"xp":-1.897}
I:Comm:> {"xp":-1.904}
I:Comm:> {"xp":-2.121}
I:Comm:> {"xp":-2.341}
I:Comm:> {"xp":-2.561}
I:Comm:> {"xp":-2.781}
I:Comm:> {"xp":-3.002}
I:Comm:> {"xp":-3.222}
I:Comm:> {"xp":-3.442}
I:Comm:> {"xp":-3.662}
I:Comm:> {"xp":-3.882}
I:Comm:> {"xp":-4.102}
I:Comm:> {"xp":-4.322}
I:Comm:> {"xp":-4.542}
I:Comm:> {"xp":-4.763}
I:Comm:> {"xp":-4.983}
I:Comm:> {"xp":-5.203}
I:Comm:> {"xp":-5.423}
I:Comm:> {"xp":-5.643}
I:Comm:> {"xp":-5.863}
I:Comm:> {"xp":-6.083}
I:Comm:> {"xp":-6.178}
I:Comm:> {"xp":-6.185}
I:Comm:> {"xp":-6.402}
I:Comm:> {"xp":-6.622}
I:Comm:> {"xp":-6.842}
I:Comm:> {"xp":-7.062}
I:Comm:> {"xp":-7.282}
I:Comm:> {"xp":-7.502}
I:Comm:> {"xp":-7.723}
I:Comm:> {"xp":-7.943}
I:Comm:> {"xp":-8.163}
I:Comm:> {"xp":-8.383}
I:Comm:> {"xp":-8.603}
I:Comm:> {"xp":-8.823}
I:Comm:> {"xp":-9.043}
I:Comm:> {"xp":-9.263}
I:Comm:> {"xp":-9.484}
I:Comm:> {"xp":-9.704}
I:Comm:> {"xp":-9.924}
I:Comm:> {"xp":-10.144}
I:Comm:> {"xp":-10.364}
I:Comm:> {"xp":-10.459}
I:Comm:> {"xp":-10.466}
I:Comm:> {"xp":-10.682}
I:Comm:> {"xp":-10.903}
I:Comm:> {"xp":-11.123}
I:Comm:> {"xp":-11.343}
I:Comm:> {"xp":-11.563}
I:Comm:> {"xp":-11.783}
I:Comm:> {"xp":-12.003}
I:Comm:> {"xp":-12.223}
I:Comm:> {"xp":-12.444}
I:Comm:> {"xp":-12.664}
I:Comm:> {"xp":-12.884}
I:Comm:> {"xp":-13.104}
I:Comm:> {"xp":-13.324}
I:Comm:> {"xp":-13.544}
I:Comm:> {"xp":-13.764}
I:Comm:> {"xp":-13.984}
I:Comm:> {"xp":-14.066}
I:Comm:> {"xp":-14.069}
I:Comm:> {"xp":-14.289}
I:Comm:> {"xp":-14.509}
I:Comm:> {"xp":-14.73}
I:Comm:> {"xp":-14.95}
I:Comm:> {"xp":-15.17}
I:Comm:> {"xp":-15.39}
I:Web:200 GET /sockjs/info?t=1615162628918 (10.0.0.17)
I:Comm:> {"xp":-15.61}
I:Comm:> {"xp":-15.827}
I:Comm:> {"xp":-16.047}
I:Web:101 GET /sockjs/181/nqsi4wnz/websocket (10.0.0.17)
I:Web:Connection from 10.0.0.17
I:Web:200 GET /api/config/load?_=1615161785644 (10.0.0.17)
I:Comm:> {"xp":-16.267}
I:Comm:> {"xp":-16.487}
I:Comm:> {"xp":-16.707}
I:Comm:> {"xp":-16.927}
I:Comm:> {"xp":-17.148}
I:Comm:> {"xp":-17.368}
I:Comm:> {"xp":-17.588}
I:Comm:> {"xp":-17.808}
I:Comm:> {"xp":-17.896}
I:Comm:> {"xp":-17.899}
I:Comm:> {"xp":-18.12}
I:Comm:> {"xp":-18.34}
I:Comm:> {"xp":-18.56}
I:Comm:> {"xp":-18.78}
I:Comm:> {"xp":-19}
I:Comm:> {"xp":-19.02}
I:Comm:> {"xp":-19.027}
I:Comm:> {"xp":-19.244}
I:Comm:> {"xp":-19.464}
I:Comm:> {"xp":-19.684}
I:Comm:> {"xp":-19.904}
I:Comm:> {"xp":-20.124}
I:Comm:> {"xp":-20.345}
I:Comm:> {"xp":-20.565}
I:Comm:> {"xp":-20.785}
I:Comm:> {"xp":-21.005}
I:Comm:> {"xp":-21.225}
I:Comm:> {"xp":-21.445}
I:Comm:> {"xp":-21.665}
I:Comm:> {"xp":-21.726}
I:Comm:> {"xp":-21.73}
I:Comm:> {"xp":-21.95}
I:Comm:> {"xp":-22.17}
I:Comm:> {"xp":-22.39}
I:Comm:> {"xp":-22.61}
I:Comm:> {"xp":-22.83}
I:Comm:> {"xp":-23.047}
I:Comm:> {"xp":-23.267}
I:Comm:> {"xp":-23.487}
I:Comm:> {"xp":-23.528}
I:Comm:> {"xp":-23.531}
I:Comm:> {"xp":-23.752}
I:Comm:> {"xp":-23.972}
I:Comm:> {"xp":-24.192}
I:Comm:> {"xp":-24.412}
I:Comm:> {"xp":-24.632}
I:Comm:> {"xp":-24.852}
I:Comm:> {"xp":-25.072}
I:Comm:> {"xp":-25.293}
I:Comm:> {"xp":-25.513}
I:Comm:> {"xp":-25.733}
I:Comm:> {"xp":-25.953}
I:Comm:> {"xp":-26.173}
I:Comm:> {"xp":-26.393}
I:Comm:> {"xp":-26.613}
I:Comm:> {"xp":-26.833}
I:Comm:> {"xp":-27.054}
I:Comm:> {"xp":-27.274}
I:Comm:> {"xp":-27.494}
I:Comm:> {"xp":-27.582}
I:Comm:> {"xp":-27.589}
I:Comm:> {"xp":-27.805}
I:Comm:> {"xp":-28.026}
I:Comm:> {"xp":-28.246}
I:Comm:> {"xp":-28.466}
I:Comm:> {"xp":-28.686}
I:Comm:> {"xp":-28.906}
I:Comm:> {"xp":-29.126}
I:Comm:> {"xp":-29.346}
I:Comm:> {"xp":-29.567}
I:Comm:> {"xp":-29.787}
I:Comm:> {"xp":-30.007}
I:Comm:> {"xp":-30.061}
I:Comm:> {"xp":-30.064}
I:Comm:> {"xp":-30.284}
I:Comm:> {"xp":-30.505}
I:Comm:> {"xp":-30.725}
I:Comm:> {"xp":-30.945}
I:Comm:> {"xp":-31.165}
I:Comm:> {"xp":-31.189}
I:Comm:> {"xp":-31.192}
I:Comm:> {"xp":-31.412}
I:Comm:> {"xp":-31.632}
I:Comm:> {"xp":-31.853}
I:Comm:> {"xp":-32.073}
I:Comm:> {"xp":-32.293}
I:Comm:> {"xp":-32.513}
I:Comm:> {"xp":-32.733}
I:Comm:> {"xp":-32.953}
I:Comm:> {"xp":-33.173}
I:Comm:> {"xp":-33.393}
I:Comm:> {"xp":-33.614}
I:Comm:> {"xp":-33.834}
I:Comm:> {"xp":-34.054}
I:Comm:> {"xp":-34.118}
I:Comm:> {"xp":-34.122}
I:Comm:> {"xp":-34.342}
I:Comm:> {"xp":-34.562}
I:Comm:> {"xp":-34.569}
I:Comm:> {"xp":-34.572}
I:Comm:> {"xp":-34.792}
I:Comm:> {"xp":-35.012}
I:Comm:> {"xp":-35.232}
I:Comm:> {"xp":-35.453}
I:Comm:> {"xp":-35.673}
I:Comm:> {"xp":-35.893}
I:Comm:> {"xp":-36.11}
I:Comm:> {"xp":-36.33}
I:Comm:> {"xp":-36.55}
I:Comm:> {"xp":-36.77}
I:Comm:> {"xp":-36.99}
I:Comm:> {"xp":-37.21}
I:Comm:> {"xp":-37.43}
I:Comm:> {"xp":-37.65}
I:Comm:> {"xp":-37.871}
I:Comm:> {"xp":-38.091}
I:Comm:> {"xp":-38.311}
I:Comm:> {"xp":-38.531}
I:Comm:> {"xp":-38.751}
I:Comm:> {"xp":-38.971}
I:Comm:> {"xp":-39.191}
I:Comm:> {"xp":-39.412}
I:Comm:> {"xp":-39.523}
I:Comm:> {"xp":-39.527}
I:Comm:> {"xp":-39.747}
I:Comm:> {"xp":-39.967}
I:Comm:> {"xp":-40.187}
I:Comm:> {"xp":-40.407}
I:Comm:> {"xp":-40.627}
I:Comm:> {"xp":-40.848}
I:Comm:> {"xp":-41.068}
I:Comm:> {"xp":-41.102}
I:Comm:> {"xp":-41.105}
I:Comm:> {"xp":-41.325}
I:Comm:> {"xp":-41.545}
I:Comm:> {"xp":-41.765}
I:Comm:> {"xp":-41.985}
I:Comm:> {"xp":-42.206}
I:Comm:> {"xp":-42.426}
I:Comm:> {"xp":-42.646}
I:Comm:> {"xp":-42.866}
I:Comm:> {"xp":-43.086}
I:Comm:> {"xp":-43.306}
I:Comm:> {"xp":-43.526}
I:Comm:> {"xp":-43.746}
I:Comm:> {"xp":-43.967}
I:Comm:> {"xp":-44.187}
I:Comm:> {"xp":-44.407}
I:Comm:> {"xp":-44.627}
I:Comm:> {"xp":-44.847}
I:Comm:> {"xp":-45.067}
I:Comm:> {"xp":-45.287}
I:Comm:> {"xp":-45.508}
I:Comm:> {"xp":-45.728}
I:Comm:> {"xp":-45.944}
I:Comm:> {"xp":-46.165}
I:Comm:> {"xp":-46.385}
I:Comm:> {"xp":-46.605}
I:Comm:> {"xp":-46.825}
I:Comm:> {"xp":-47.045}
I:Comm:> {"xp":-47.265}
I:Comm:> {"xp":-47.485}
I:Comm:> {"xp":-47.706}
I:Comm:> {"xp":-47.926}
I:Comm:> {"xp":-48.146}
I:Comm:> {"xp":-48.366}
I:Comm:> {"xp":-48.586}
I:Comm:> {"xp":-48.806}
I:Comm:> {"xp":-48.986}
I:Comm:> {"xp":-48.992}
I:Comm:> {"xp":-49.209}
I:Comm:> {"xp":-49.429}
I:Comm:> {"xp":-49.649}
I:Comm:> {"xp":-49.87}
I:Comm:> {"xp":-50.09}
I:Comm:> {"xp":-50.31}
I:Comm:> {"xp":-50.53}
I:Comm:> {"xp":-50.75}
I:Comm:> {"xp":-50.97}
I:Comm:> {"xp":-51.19}
I:Comm:> {"xp":-51.411}
I:Comm:> {"xp":-51.631}
I:Comm:> {"xp":-51.851}
I:Comm:> {"xp":-51.915}
I:Comm:> {"xp":-51.919}
I:Comm:> {"xp":-52.139}
I:Comm:> {"xp":-52.359}
I:Comm:> {"xp":-52.579}
I:Comm:> {"xp":-52.799}
I:Comm:> {"xp":-53.019}
I:Comm:> {"xp":-53.239}
I:Comm:> {"xp":-53.459}
I:Comm:> {"xp":-53.68}
I:Comm:> {"xp":-53.717}
I:Comm:> {"xp":-53.724}
I:Comm:> {"xp":-53.94}
I:Comm:> {"xp":-54.16}
I:Comm:> {"xp":-54.381}
I:Comm:> {"xp":-54.601}
I:Comm:> {"xp":-54.821}
I:Comm:> {"xp":-55.041}
I:Comm:> {"xp":-55.072}
I:Comm:> {"xp":-55.075}
I:Comm:> {"xp":-55.295}
I:Comm:> {"xp":-55.515}
I:Comm:> {"xp":-55.735}
I:Comm:> {"xp":-55.952}
I:Comm:> {"xp":-56.172}
I:Comm:> {"xp":-56.392}
I:Comm:> {"xp":-56.612}
I:Comm:> {"xp":-56.833}
I:Comm:> {"xp":-57.053}
I:Comm:> {"xp":-57.273}
I:Comm:> {"xp":-57.493}
I:Comm:> {"xp":-57.713}
I:Comm:> {"xp":-57.933}
I:Comm:> {"xp":-58.153}
I:Comm:> {"xp":-58.224}
I:Comm:> {"xp":-58.228}
I:Comm:> {"xp":-58.448}
I:Comm:> {"xp":-58.668}
I:Comm:> {"xp":-58.888}
I:Comm:> {"xp":-59.108}
I:Comm:> {"xp":-59.329}
I:Comm:> {"xp":-59.352}
I:Comm:> {"xp":-59.356}
I:Comm:> {"xp":-59.576}
I:Comm:> {"xp":-59.796}
I:Comm:> {"xp":-60.016}
I:Comm:> {"xp":-60.233}
I:Comm:> {"xp":-60.453}
I:Comm:> {"xp":-60.673}
I:Comm:> {"xp":-60.893}
I:Comm:> {"xp":-61.113}
I:Comm:> {"xp":-61.333}
I:Comm:> {"xp":-61.554}
I:Comm:> {"xp":-61.774}
I:Comm:> {"xp":-61.994}
I:Comm:> {"xp":-62.214}
I:Comm:> {"xp":-62.434}
I:Comm:> {"xp":-62.654}
I:Comm:> {"xp":-62.874}
I:Comm:> {"xp":-63.095}
I:Comm:> {"xp":-63.315}
I:Comm:> {"xp":-63.535}
I:Comm:> {"xp":-63.755}
I:Comm:> {"xp":-63.975}
I:Comm:> {"xp":-64.195}
I:Comm:> {"xp":-64.415}
I:Comm:> {"xp":-64.635}
I:Comm:> {"xp":-64.856}
I:Comm:> {"xp":-65.076}
I:Comm:> {"xp":-65.296}
I:Comm:> {"xp":-65.516}
I:Comm:> {"xp":-65.658}
I:Comm:> {"xp":-65.662}
I:Comm:> {"xp":-65.882}
I:Comm:> {"xp":-66.102}
I:Comm:> {"xp":-66.322}
I:Comm:> {"xp":-66.336}
I:Comm:> {"xp":-66.339}
I:Comm:> {"xp":-66.559}
I:Comm:> {"xp":-66.779}
I:Comm:> {"xp":-66.999}
I:Comm:> {"xp":-67.219}
I:Comm:> {"xp":-67.44}
I:Comm:> {"xp":-67.66}
I:Comm:> {"xp":-67.88}
I:Comm:> {"xp":-68.1}
I:Comm:> {"xp":-68.32}
I:Comm:> {"xp":-68.54}
I:Comm:> {"xp":-68.76}
I:Comm:> {"xp":-68.981}
I:Comm:> {"xp":-69.201}
I:Comm:> {"xp":-69.421}
I:Comm:> {"xp":-69.641}
I:Comm:> {"xp":-69.861}
I:Comm:> {"xp":-70.081}
I:Comm:> {"xp":-70.166}
I:Comm:> {"xp":-70.169}
I:Comm:> {"xp":-70.389}
I:Comm:> {"xp":-70.61}
I:Comm:> {"xp":-70.83}
I:Comm:> {"xp":-71.05}
I:Comm:> {"xp":-71.27}
I:Comm:> {"xp":-71.49}
I:Comm:> {"xp":-71.517}
I:Comm:> {"xp":-71.521}
I:Comm:> {"xp":-71.741}
I:Comm:> {"xp":-71.961}
I:Comm:> {"xp":-72.181}
I:Comm:> {"xp":-72.401}
I:Comm:> {"xp":-72.621}
I:Comm:> {"xp":-72.645}
I:Comm:> {"xp":-72.648}
I:Comm:> {"xp":-72.868}
I:Comm:> {"xp":-73.089}
I:Comm:> {"xp":-73.305}
I:Comm:> {"xp":-73.525}
I:Comm:> {"xp":-73.746}
I:Comm:> {"xp":-73.966}
I:Comm:> {"xp":-74.186}
I:Comm:> {"xp":-74.406}
I:Comm:> {"xp":-74.626}
I:Comm:> {"xp":-74.846}
I:Comm:> {"xp":-75.066}
I:Comm:> {"xp":-75.287}
I:Comm:> {"xp":-75.507}
I:Comm:> {"xp":-75.727}
I:Comm:> {"xp":-75.798}
I:Comm:> {"xp":-75.801}
I:Comm:> {"xp":-76.021}
I:Comm:> {"xp":-76.242}
I:Comm:> {"xp":-76.462}
I:Comm:> {"xp":-76.682}
I:Comm:> {"xp":-76.902}
I:Comm:> {"xp":-77.122}
I:Comm:> {"xp":-77.342}
I:Comm:> {"xp":-77.562}
I:Comm:> {"xp":-77.783}
I:Comm:> {"xp":-77.827}
I:Comm:> {"xp":-77.83}
I:Comm:> {"xp":-78.05}
I:Comm:> {"xp":-78.27}
I:Comm:> {"xp":-78.49}
I:Comm:> {"xp":-78.71}
I:Comm:> {"xp":-78.931}
I:Comm:> {"xp":-79.151}
I:Comm:> {"xp":-79.367}
I:Comm:> {"xp":-79.588}
I:Comm:> {"xp":-79.808}
I:Comm:> {"xp":-79.852}
I:Comm:> {"xp":-79.859}
I:Comm:> {"xp":-80.075}
I:Comm:> {"xp":-80.295}
I:Comm:> {"xp":-80.302}
I:Comm:> {"xp":-80.309}
I:Comm:> {"xp":-80.526}
I:Comm:> {"xp":-80.746}
I:Comm:> {"xp":-80.966}
I:Comm:> {"xp":-81.186}
I:Comm:> {"xp":-81.406}
I:Comm:> {"xp":-81.626}
I:Comm:> {"xp":-81.847}
I:Comm:> {"xp":-82.067}
I:Comm:> {"xp":-82.287}
I:Comm:> {"xp":-82.507}
I:Comm:> {"xp":-82.727}
I:Comm:> {"xp":-82.947}
I:Comm:> {"xp":-83.167}
I:Comm:> {"xp":-83.387}
I:Comm:> {"xp":-83.608}
I:Comm:> {"xp":-83.828}
I:Comm:> {"xp":-84.048}
I:Comm:> {"xp":-84.268}
I:Comm:> {"xp":-84.488}
I:Comm:> {"xp":-84.708}
I:Comm:> {"xp":-84.928}
I:Comm:> {"xp":-85.033}
I:Comm:> {"xp":-85.04}
I:Comm:> {"xp":-85.26}
I:Comm:> {"xp":-85.477}
I:Comm:> {"xp":-85.697}
I:Comm:> {"xp":-85.917}
I:Comm:> {"xp":-86.137}
I:Comm:> {"xp":-86.358}
I:Comm:> {"xp":-86.578}
I:Comm:> {"xp":-86.798}
I:Comm:> {"xp":-87.018}
I:Comm:> {"xp":-87.062}
I:Comm:> {"xp":-87.065}
I:Comm:> {"xp":-87.285}
I:Comm:> {"xp":-87.506}
I:Comm:> {"xp":-87.726}
I:Comm:> {"xp":-87.946}
I:Comm:> {"xp":-88.166}
I:Comm:> {"xp":-88.386}
I:Comm:> {"xp":-88.606}
I:Comm:> {"xp":-88.826}
I:Comm:> {"xp":-89.047}
I:Comm:> {"xp":-89.267}
I:Comm:> {"xp":-89.487}
I:Comm:> {"xp":-89.707}
I:Comm:> {"xp":-89.768}
I:Comm:> {"xp":-89.771}
I:Comm:> {"xp":-89.991}
I:Comm:> {"xp":-90.212}
I:Comm:> {"xp":-90.218}
I:Comm:> {"xp":-90.222}
I:Comm:> {"xp":-90.442}
I:Comm:> {"xp":-90.662}
I:Comm:> {"xp":-90.882}
I:Comm:> {"xp":-91.102}
I:Comm:> {"xp":-91.319}
I:Comm:> {"xp":-91.539}
I:Comm:> {"xp":-91.759}
I:Comm:> {"xp":-91.793}
I:Comm:> {"xp":-91.8}
I:Comm:> {"xp":-92.017}
I:Comm:> {"xp":-92.237}
I:Comm:> {"xp":-92.457}
I:Comm:> {"xp":-92.677}
I:Comm:> {"xp":-92.897}
I:Comm:> {"xp":-92.921}
I:Comm:> {"xp":-92.924}
I:Comm:> {"xp":-93.144}
I:Comm:> {"xp":-93.365}
I:Comm:> {"xp":-93.585}
I:Comm:> {"xp":-93.805}
I:Comm:> {"xp":-94.025}
I:Comm:> {"xp":-94.245}
I:Comm:> {"xp":-94.465}
I:Comm:> {"xp":-94.685}
I:Comm:> {"xp":-94.905}
I:Comm:> {"xp":-94.95}
I:Comm:> {"xp":-94.953}
I:Comm:> {"xp":-95.173}
I:Comm:> {"xp":-95.393}
I:Comm:> {"xp":-95.613}
I:Comm:> {"xp":-95.833}
I:Comm:> {"xp":-96.054}
I:Comm:> {"xp":-96.274}
I:Comm:> {"xp":-96.494}
I:Comm:> {"xp":-96.711}
I:Comm:> {"xp":-96.931}
I:Comm:> {"xp":-96.975}
I:Comm:> {"xp":-96.982}
I:Comm:> {"xp":-97.198}
I:Comm:> {"xp":-97.418}
I:Comm:> {"xp":-97.639}
I:Comm:> {"xp":-97.859}
I:Comm:> {"xp":-98.079}
I:Comm:> {"xp":-98.299}
I:Comm:> {"xp":-98.519}
I:Comm:> {"xp":-98.739}
I:Comm:> {"xp":-98.959}
I:Comm:> {"xp":-99.179}
I:Comm:> {"xp":-99.4}
I:Comm:> {"xp":-99.62}
I:Comm:> {"xp":-99.84}
I:Comm:> {"xp":-100.06}
I:Comm:> {"xp":-100.28}
I:Comm:> {"xp":-100.355}
I:Comm:> {"xp":-100.361}
I:Comm:> {"xp":-100.578}
I:Comm:> {"xp":-100.798}
I:Comm:> {"xp":-101.018}
I:Comm:> {"xp":-101.239}
I:Comm:> {"xp":-101.459}
I:Comm:> {"xp":-101.679}
I:Comm:> {"xp":-101.899}
I:Comm:> {"xp":-102.119}
I:Comm:> {"xp":-102.339}
I:Comm:> {"xp":-102.559}
I:Comm:> {"xp":-102.779}
I:Comm:> {"xp":-103}
I:Comm:> {"xp":-103.22}
I:Comm:> {"xp":-103.44}
I:Comm:> {"xp":-103.66}
I:Comm:> {"xp":-103.88}
I:Comm:> {"xp":-104.1}
I:Comm:> {"xp":-104.32}
I:Comm:> {"xp":-104.412}
I:Comm:> {"xp":-104.415}
I:Comm:> {"xp":-104.635}
I:Comm:> {"xp":-104.856}
I:Comm:> {"xp":-105.076}
I:Comm:> {"xp":-105.296}
I:Comm:> {"xp":-105.516}
I:Comm:> {"xp":-105.736}
I:Comm:> {"xp":-105.956}
I:Comm:> {"xp":-106.176}
I:Comm:> {"xp":-106.396}
I:Comm:> {"xp":-106.617}
I:Comm:> {"xp":-106.837}
I:Comm:> {"xp":-107.057}
I:Comm:> {"xp":-107.277}
I:Comm:> {"xp":-107.497}
I:Comm:> {"xp":-107.717}
I:Comm:> {"xp":-107.937}
I:Comm:> {"xp":-108.158}
I:Comm:> {"xp":-108.378}
I:Comm:> {"xp":-108.598}
I:Comm:> {"xp":-108.818}
I:Comm:> {"xp":-109.038}
I:Comm:> {"xp":-109.258}
I:Comm:> {"xp":-109.478}
I:Comm:> {"xp":-109.695}
I:Comm:> {"xp":-109.915}
I:Comm:> {"xp":-110.135}
I:Comm:> {"xp":-110.271}
I:Comm:> {"xp":-110.274}
I:Comm:> {"xp":-110.494}
I:Comm:> {"xp":-110.714}
I:Comm:> {"xp":-110.935}
I:Comm:> {"xp":-110.945}
I:Comm:> {"xp":-110.952}
I:Comm:> {"0sl":false,"xp":110.78}
I:Comm:> {"0sl":true,"xp":110.56}
I:Comm:> {"xp":110.34}
I:Comm:> {"xp":110.327}
I:Comm:> {"xp":110.323}
I:Comm:> {"xp":110.103}
I:Comm:> {"xp":109.883}
I:Comm:> {"xp":109.663}
I:Comm:> {"xp":109.443}
I:Comm:> {"xp":109.222}
I:Comm:> {"xp":109.002}
I:Comm:> {"xp":108.782}
I:Comm:> {"xp":108.562}
I:Comm:> {"xp":108.342}
I:Comm:> {"xp":108.122}
I:Comm:> {"xp":107.902}
I:Comm:> {"xp":107.682}
I:Comm:> {"xp":107.461}
I:Comm:> {"xp":107.241}
I:Comm:I2C: S b=None w=None d=None
I:AVR:I2C: S b=None w=None d=None
I:Web:200 PUT /api/stop (10.0.0.17)
****************************

**Comment by jcoffland** [comment]
I found and fixed this problem.  There was a numerical overflow.  You can try out the fix in the [v1.0.0-rc5 beta firmware](https://buildbotics.com/bbctrl/bbctrl-1.0.0-rc5.tar.bz2).  Note, this firmware release is not complete and has some other know problems.  However, these problems should be limited to the new web interface.

**Comment by mmcmilli** [comment]
That's great!

**Comment by DougCoffland** [comment]
Resolved in V1.0.0
